hedyorg/hedy
# 🪲 Broken boolean translation

Fixes #5572
Fixes tests failing due to boolean translation

**How to test**
1. Ensure that tests are failing: checkout the **main** branch, comment out the code in snippet_already_tested_with_current_hedy_version and make it always return False, run the tests test_level_15.py and note that tests are failing.
2. To check that the error are fixed: checkout the boolean_translation_5572 branch, again make the snippet_already_tested_with_current_hedy_version always return False and run all tests, ensure that they are all successful.
